consensus/core/src/utils.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::types::AuthorityRound;
+
+use consensus_config::AuthorityIndex;
+
+fn format_authority_index(i: AuthorityIndex) -> char {
+    ('A' as usize + i.value()) as u8 as char
```

Comment:
When we have more than 26 nodes this method is rolling over to other symbols and honestly on the various environment experiments it made debugging actually harder. If we consider it useful for local test debugging , when having a few nodes it is ok. Otherwise we can just fallback to the index number.